aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -1571,6 +1667,16 @@ module aptos_framework::delegation_pool {
                 pool.observed_lockup_cycle,
                 pool_u64::create_with_scaling_factor(SHARES_SCALING_FACTOR)
             );
+        };
+
+        if (exists<NextCommissionPercentage>(pool_address)) {
+            let next_commission_percentage = borrow_global<NextCommissionPercentage>(pool_address);
+            if (timestamp::now_seconds() >= next_commission_percentage.effective_after_secs) {
+                // update the commission percentage for the new lockup cycle
+                pool.operator_commission_percentage = borrow_global<NextCommissionPercentage>(
```

Comment:
use `next_commission_percentage` here

Reply:
Updated. Couldn't use `operator_commission_percentage_next_lockup_cycle(pool_address)` which causes a dangling reference error because `pool` is already mutably borrowed.